src/commands/createFunction/FunctionCreateStepBase.ts

```diff
@@ -5,13 +5,16 @@
 
 import { AzExtFsExtra, AzureWizardExecuteStep, callWithTelemetryAndErrorHandling, IActionContext } from '@microsoft/vscode-azext-utils';
 import { Progress, Uri, window, workspace } from 'vscode';
+import { hostFileName } from '../../constants';
 import { ext } from '../../extensionVariables';
+import { IHostJsonV2 } from '../../funcConfig/host';
 import { localize } from '../../localize';
 import { IFunctionTemplate } from '../../templates/IFunctionTemplate';
 import { nonNullProp } from '../../utils/nonNull';
 import { verifyExtensionBundle } from '../../utils/verifyExtensionBundle';
 import { getContainingWorkspace } from '../../utils/workspace';
 import { IFunctionWizardContext } from './IFunctionWizardContext';
+import path = require('path');
```

Comment:
Does this not work?
```suggestion
import * as path from 'path';
```
If it does work, ESLint will also want to reorder the imports.